src/System.Private.ServiceModel/tests/Common/Unit/ServiceInterfaces.cs

```diff
@@ -175,15 +175,3 @@ public interface IWcfDuplexTaskReturnCallback
     Task<Guid> ServicePingFaultCallback(Guid guid);
 }
 
-[ServiceContract]
```

Comment:
Reviewers: this code was both dead and redundant with the same definition elsewhere.  Removed for cleanup.